bundle/src/files.rs

```diff
@@ -48,6 +51,7 @@ impl FileSet {
     pub fn scan_from_glob(
         repo_root: &str,
         glob_path: String,
+        test_runner_status: Option<TestRunnerJunitStatus>,
```

Comment:
Feels like we should just accept `JunitPathWrapper` here instead of making these two separate arguments